packages/event/src/Charcoal/Event/InterruptableEventInterface.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace Charcoal\Event;
+
+use Stringable;
+
+/**
+ * Interface: InterruptableEventInterface
+ * @package Charcoal\Event
+ */
+interface InterruptableEventInterface
+{
+    public function isInterrupted(): bool;
+
+    /**
+     * @return string|Stringable
+     */
+    public function reason();
```

Comment:
What if:

```suggestion
    public function getReason();
```

or

```suggestion
    public function getReasonForInterruption();
```

?